spec/models/miq_expression_spec.rb

```diff
@@ -2020,7 +2034,7 @@
 
   describe "#field_in_sql?" do
     it "returns false for model.virtualfield" do
```

Comment:
I changed the column to be a virtual column with no sql.

change description to "virtual field (with no sql)